src/functions.php

```diff
@@ -60,7 +60,7 @@ function describe_type($input)
  *                     format: "Name: Value"
  * @return array
  */
-function headers_from_lines($lines)
```

Comment:
Currently this can take any iterable like a `Generator` etc. Adding the array type will be a BC break

Reply:
I fixed this